src/vos/vos_common.c

```diff
@@ -457,7 +457,11 @@ vos_mod_init(void)
 	if (vos_start_epoch == DAOS_EPOCH_MAX)
 		vos_start_epoch = d_hlc_get();
 
-	rc = vos_pool_settings_init();
+	/**
+	 * Pass MD mode as int to enable custom umem allocator to be requested e.g. PMEM (0),
+	 * BMEM (1) and ADMEM (2). For MD on SSD specify (1) for now.
+	 */
```

Comment:
Then we won't be able to enable ADMEM. I think we should retain the DAOS_MD_ON_SSD env var to specify ADMEM or BMEM (the env var name could be changed).

Reply:
done, name changed to "DAOS_MD_ON_SSD_MODE"